canonical-web-and-design/vanilla-framework
# Code snippet pattern

## Done

Added new `p-code-snippet` pattern, with two child components: `p-code-snippet__header` and `p-code-snippet__block`.

Fixes #3474 

## QA

- Open [demo](https://vanilla-framework-3474.demos.haus/docs/examples/patterns/code-snippet/code-snippet)
- See that the design matches the one in Zeplin (though without a dropdown, which is coming in a separate PR for #3469): https://app.zeplin.io/project/5fa270d8eee4dbb267e579f1/screen/5fc616c95320fdb32885432e
